### PR TITLE
Use new radar-output-restructure

### PR DESCRIPTION
**Pull request by blootsvoets**
Use updated radar-output-restructure as detailed in RADAR-base/Restructure-HDFS-Topic#58. Pending acceptance and properly published docker image tag.

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml [hunk 1]
```diff
@@ -826,3 +832,15 @@ services:
...
       RADAR_HDFS_RESTRUCTURE_OPTS: -Xms250m -Xmx4g
     command: -F /etc/restructure.yml -S
+
+  output-redis:
+    image: bitnami/redis:5.0.8
+    volumes:
+      - redis-data:/data
+    networks:
+      - redis
+    command:
```
Comment (nivemaham) [inline dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml:842]: I think the command part is not necessary for redis.

Comment (blootsvoets) [reply dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml]: Yes, you're right. I mistakingly copied it from the minio container.

File: dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml [hunk 2]
```diff
@@ -826,3 +832,12 @@ services:
     environment:
       RADAR_HDFS_RESTRUCTURE_OPTS: -Xms250m -Xmx4g
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml:833]: These are JAVA_OPTS now? Also, we can copy the other JAVA_OPTS mentioned in the Dockerfile

Comment (blootsvoets) [reply dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml:833]: No please do not use `JAVA_OPTS`, use `RADAR_OUTPUT_RESTRUCTURE_OPTS` instead.

Comment (yatharthranjan) [reply dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml:833]: ok thanks

**Pull request merged**